### PR TITLE
Update jQuery.validation to 1.19.0

### LINKED ISSUE #272: Have the JS URL validator validate valid URLs
At current a valid (although not necessarily _complete_) URL will be shown as invalid
image

However adding a protocol to it will see the validation pass
image

This issue only affects the JavaScript validator - disabling JavaScript and then submitting the form sees a successful submission (although in this case I feel that any value may be valid - although that should be a separate issue _if true_).

### PR DESCRIPTION
**Pull request by NightJar**
In an attempt to fix #272 I've updated the validation jQuery plug-in.
It did not solve the issue, but updating is never a bad idea anyway.

### CODE REVIEW

**Comment by ScopeyNZ** [comment]
Is there a way for us to just take the `dist` files? This PR seems to include a lot of unnecessary extra files (like demos).

**Comment by NightJar** [comment]
I could whittle it down I guess, I personally find it "more obvious" to include the full dist (i.e. release package, not `min` file), and is a pattern seen elsewhere with our modules.

File: src/Extensions/CommentsExtension.php [hunk 1]
```diff
@@ -463,7 +463,7 @@ public function CommentsForm()
         $enabled = $this->getCommentsEnabled();
         if ($enabled && $this->owner->getCommentsOption('include_js')) {
             Requirements::javascript('//code.jquery.com/jquery-3.3.1.min.js');
-            Requirements::javascript('silverstripe/comments:thirdparty/jquery-validate/jquery.validate.min.js');
+            Requirements::javascript('silverstripe/comments:thirdparty/jquery-validation/jquery.validate.min.js');
```
Comment (robbieaverill) [inline src/Extensions/CommentsExtension.php]: Even though third party libraries aren't part of our public API, this path name change might cause problems for anyone relying on those files to exist and would introduce unnecessary risk for a patch release. Can you rename it back to what it was?

Comment (NightJar) [reply src/Extensions/CommentsExtension.php]: huh, yeah. Strange the library (as per filename) is jquery.validate - but the folder name came direct from release distribution; 'jquery-validation'. Weird.

Sure I'll put it back.

Comment (NightJar) [reply src/Extensions/CommentsExtension.php]: I have resolved this comment, but I'll let you resolve this conversation :)

**Pull request merged**